javascripts/govuk/analytics/google-analytics-universal-tracker.js

```diff
@@ -33,24 +34,25 @@
 
   // https://developers.google.com/analytics/devguides/collection/analyticsjs/pages
   GoogleAnalyticsUniversalTracker.prototype.trackPageview = function(path, title, options) {
-    var options = options || {};
+    var options;
```

Comment:
Good point. I saw it as a simplification, but missed that you were re-declaring `options`.

Let's tidy this up.